fusion vs docs (#8814)
fishtown-analytics/docs.getdbt.com: website/docs/guides/fusion-qs.md

    ```bash
    dbtf build
    ```

>     With the <Constant name="fusion_engine" />, generate docs metadata in <Constant name="platform_cli" /> with `dbtf build --write-catalog`; if you use dbt platform, you can browse that metadata in [<Constant name="catalog" />](/docs/explore/build-and-view-your-docs).

Author: ```suggestion
    With the <Constant name="fusion_engine" />, generate docs metadata in <Constant name="platform_cli" /> with `dbtf build --write-catalog`; if you use <Constant name="dbt_platform" />, you can browse that metadata in [<Constant name="catalog" />](/docs/explore/build-and-view-your-docs).
```

>     After running `dbtf build`, use [<Constant name="catalog" />](/docs/explore/build-and-view-your-docs) to browse your project documentation. With the <Constant name="fusion_engine" />, generate docs metadata using [`--write-catalog`](/reference/commands/cmd-docs?version=2.0#--write-catalog-flag) instead of `dbt docs generate`.

Reviewer: i'm confused with this: is this guide for dbt platform users? it reads like you're saying users should go to catalog after dbtf build. it sound like dbtf build generates metadata by default.

and then you're saying / implying that users should --write-catalog instead of dbt docs generate. 

adding this here also doesn't flow with lines 150-153.

>     After running `dbtf build`, use [<Constant name="catalog" />](/docs/explore/build-and-view-your-docs) to browse your project documentation. With the <Constant name="fusion_engine" />, generate docs metadata using [`--write-catalog`](/reference/commands/cmd-docs?version=2.0#--write-catalog-flag) instead of `dbt docs generate`.

Author: ```suggestion
    With the <Constant name="fusion_engine" />, generate docs metadata in <Constant name="platform_cli" /> with `dbtf build --write-catalog`; if you use dbt platform, you can browse that metadata in [<Constant name="catalog" />](/docs/explore/build-and-view-your-docs).
```
